### ISSUE TITLE
Drupal 8 Zend OPcache support

### ISSUE THREAD

**Issue by jkenn99** [omega8cc/boa#847]
The default PHP configuration disables `opcache_get_status()` and `opcache_invalidate()` both of which are being used by d8 core. What is the reason for disabling these functions? What is the best way to safely re-enable them for our d8 platforms?

**Comment by omega8cc** [comment]
It is a bit problematic. We have very restrictive configuration for security reasons. Some opcache functions, like `opcache_get_status()` could be used to reveal paths between sites and Octopus instances. The `opcache_get_status()` is not needed for anything in the opcache itself, and the only reason to allow it, would be to get rid of that warning added in D8. But then we can't use `opcache.restrict_api` protection. I think we will patch D8 core to get rid of that confusing warning and check for any other available `opcache_` function instead, and we will remove only `opcache_invalidate()` from the blacklist, since it should be safe to use.

**Comment by omega8cc** [comment]
By the way, Symphony does this far better and Drupal should re-use this instead of introducing too demanding requirements on its own:

```
'zend_opcache_enabled' => extension_loaded('Zend OPcache') && ini_get('opcache.enable'),
```

**Comment by omega8cc** [comment]
I have reopened related issue on d.o: https://www.drupal.org/node/2502507#comment-10693520

**Comment by omega8cc** [comment]
And the patch: https://www.drupal.org/node/2639878

**Comment by omega8cc** [comment]
Looks like it will be fixed in D8 core soon: https://www.drupal.org/node/2639878

**Comment by omega8cc** [comment]
Just to confirm: the current D8 message is a false alarm. It's using wrong test while opcache does work.

**Comment by omega8cc** [comment]
The fix has been committed to 8.2.x -- https://www.drupal.org/node/2639878#comment-11459899
